src/wizard/interactive_build.jl

```diff
@@ -184,6 +184,60 @@ function diff_srcdir(state::WizardState, prefix::Prefix, ur::Runner)
     return false
 end
 
+function bb_add(client, state::WizardState, prefix::Prefix, platform::Platform, jll::AbstractString)
+    if any(dep->getpkg(dep).name == jll, state.dependencies)
+        println(client, "ERROR: Package was already added")
+        return
+    end
+    new_dep = Dependency(jll)
+    try
+        # This will redo some work, but that may be ok
+        setup_dependencies(prefix, getpkg.([state.dependencies; new_dep]), platform)
+        push!(state.dependencies, new_dep)
+    catch e
+        showerror(client, e)
```

Comment:
Is this a sufficiently helpful error message when `jll` doesn't exist in the registry?

Reply:
Yes, you get the same error message you would as if you tried to `Pkg.add` it.